kde-apps/kdeedu-data/kdeedu_data_kf6-24.12.3.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Files for educational applications"
+DESCRIPTION="Shared icons, artwork and data files for educational applications"
+HOMEPAGE="https://invent.kde.org/education/kdeedu-data"
+COPYRIGHT="2010-2025 KDE Organisation"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://download.kde.org/stable/release-service/$portVersion/src/kdeedu-data-$portVersion.tar.xz"
+CHECKSUM_SHA256="2961d58f7d80e38babe724bb7225369d1619c54d28b3bd05d8dcd333be9d666f"
+SOURCE_DIR="kdeedu-data-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	kdeedu_data_kf6$secondaryArchSuffix = $portVersion
+	"
+if [ "$targetArchitecture" = "x86_gcc2" ]; then
+	PROVIDES+="
+		kdeedu_data_kf6 = $portVersion
+		"
+fi
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	extra_cmake_modules$secondaryArchSuffix
+	devel:libQt6Core$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
```

Comment:
Why does this require gcc if it contains no binaries?

Reply:
Without it I'm getting the next error:
```
CMake Error at CMakeLists.txt:3 (project):
  No CMAKE_C_COMPILER could be found.

  Tell CMake where to find the compiler by setting either the environment
  variable "CC" or the CMake cache entry CMAKE_C_COMPILER to the full path to
  the compiler, or to the compiler name if it is in the PATH.


CMake Error at CMakeLists.txt:3 (project):
  No CMAKE_CXX_COMPILER could be found.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.
```